src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -163,6 +164,13 @@ const NameCell = styled(Cell)`
   padding: 0px 8px;
   min-width: 240px;
   gap: 8px;
+
+  @media only screen and (max-width: ${XSMALL_MEDIA_BREAKPOINT}) {
```

Comment:
Ahh the reason I opted for using the `XSMALL_MEDIA_BREAKPOINT` is because if we use the `SMALL_MEDIA_BREAKPOINT`, then on non-mini phones we sometimes have the issue of shortening text when we don't need to as observed in the ss you attached for wrapped liquid eth.

<img width="390" alt="Screenshot 2023-07-19 at 2 56 47 PM" src="https://github.com/Uniswap/interface/assets/35351983/f20a2aa1-c976-422e-b1c5-d0a58ccb80e1">

Here's how it looks on main now:
<img width="359" alt="Screenshot 2023-07-19 at 2 58 39 PM" src="https://github.com/Uniswap/interface/assets/35351983/2dbb16e6-cc66-488b-99a1-52acbff845a2">

I just thought it would be better if we displayed as much as possible if we didn't need to shorten text. Lmk your thoughts

Reply:
Have you looked into setting a max-width being a %age of the parent element? Something like 50-60% would allow for more text to show on mini phones as well
![Screenshot 2023-07-20 at 9 10 48 AM](https://github.com/Uniswap/interface/assets/11512321/4b23f8bc-a38f-4d84-bb04-c1d7f1f67814)

Reply:
This would remove the issue with force shortening of text for screens 360 - 396px

Reply:
ahh yeah I tried that and it looked better! after doing some more testing, I think just decreasing the `min-width` to `200px` after the breakpoint seems to look the best. Let me know your thoughts